forum/users/serializers.py

```diff
@@ -56,5 +61,19 @@ def create(self, validated_data):
         user = User.objects.create_user(**validated_data)
         if roles_data:
             user.roles.set(roles_data)
+            
+        token = RefreshToken.for_user(user).access_token
+        
+        
+        activation_url = f"http://localhost:8000{reverse('activate', kwargs={'token': str(token)})}"
```

Comment:
You are using the `RefreshToken.for_user(user).access_token` to generate the activation token. However, this is a short-lived token by default. Since activation is typically not time-sensitive, consider using a longer-lived token, like a custom token or a different payload.
You could create a dedicated activation token (a custom token) or configure the lifespan of the activation token in your JWT settings.
```
token = AccessToken.for_user(user)
activation_url = f"http://localhost:8000{reverse('activate', kwargs={'token': str(token)})}"
```